src/main/game/entities/Entity.java

```diff
@@ -8,18 +8,28 @@
 import game.visitors.iTileActionVisitor;
 
 public abstract class Entity {
+
     protected PowerState powerState;
     protected Queue<Command> commandQueue;
     protected int health;
     protected HealthPercentage healthPercent;
     private EntityId entityId;
     protected PlacementManager placementManager;
+    protected DeathNotifier notifer;
 
-    //TODO Get movementManager into constructor
+    // Constructor
     public Entity(EntityId entityId, PlacementManager povementManager){
         this.commandQueue = new LinkedList<>();
         this.entityId = entityId;
-        this.placementManager=placementManager;
+        this.placementManager = placementManager;
+    }
+
+    // Constructor w/ DN
+    public Entity(EntityId entityId, PlacementManager povementManager, DeathNotifier notifier) {
```

Comment:
Change povementManager to placementManager. Don't worry about the one before I already did it in my branch